controllers/provider-gcp/pkg/internal/infrastructure/terraform.go

```diff
@@ -144,8 +144,31 @@ func RenderTerraformerChart(
 	config *api.InfrastructureConfig,
 	cluster *extensionscontroller.Cluster,
 ) (*TerraformFiles, error) {
+	var (
+		aggregationInterval string
+		flowSampling        float32
+		metadata            string
```

Comment:
Looking at the code below, I don't get the purpose of these variables. I don't think they are needed at all.